core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -3022,10 +3022,6 @@ private void checkLiteral2(String expression, String expected) {
             + "WHERE (hire_date + INTERVAL 10 HOUR)"
             + " > TIMESTAMP '2005-10-17 00:00:00'";
     sql(sql1).withBigQuery().ok(expect1);
-
-    final String sql2 = "select  * from \"employee\" where  \"hire_date\" + "
-            + "INTERVAL '1 2:34:56.78' DAY TO SECOND > TIMESTAMP '2005-10-17 00:00:00' ";
-    sql(sql2).withBigQuery().throws_("Only INT64 is supported as the interval value for BigQuery.");
```

Comment:
Could you provide more context on the deletion of this test cases?

Reply:
That wasn't my change actually, not sure why it shows up in the diff. It was made in this Pr: https://github.com/googleinterns/calcite/pull/207